### PR TITLE
EV-1744: Documentation for Buildings and BuildingAccessAppointment API Endpoints

### PR DESCRIPTION
**Pull request by klondikemarlen**
[EV-1744](https://proof-tech.atlassian.net/browse/EV-1744)

### Overview

Add documentation for Return to Office Building API endpoints
Add documentation for Return to Office BuildingAccessAppointment API endpoints

### Solution

Wrote some markdown files with CRUD style using axios.

### Screenshots

image
image

### Testing Instructions

1. Run the instructions.

### Pre-Merge Checklist

- [x] The work here reflects the user stories in the JIRA ticket(s) listed at the top of this pull request
- [x] This pull request has been referenced via URL in the JIRA ticket(s) listed
- [x] I have performed a self-review of the code I have written here
- [x] I have done end-to-end QA testing of sections of the application I have altered
- [x] My changes generate no new warnings / errors in the console
- [x] I have demoed the code and gotten a review from a member of the CS team (for customer facing code)
- [x] I am prepared for this code to go to production today (or there is a WIP label on the pull request)
- [x] I am prepared to write hotfixes or rollback my code if something goes wrong

Reminder that deploy instructions are [here](https://github.com/proofgov/proofgov/wiki/Deploy-steps)

### CODE REVIEW

**Comment by wxgeorge** [comment]
See also https://github.com/proofgov/api/pull/5 ...

File: Return to Office/Building Access Appointments.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+    "building_access_appointment": {
+      'building_id': '<integer not null>',
+      'floor': '<string/integer? not null>',
+      'end_at': '<datetime not null>',
+      'start_at': '<datetime not null>',
+      'user_id': '<integer not null>',
+      'recurring_appointment_profile_id': '<integer>',
+      'routing_id': '<integer',
+    }
+  }
```
Comment (wxgeorge) [inline Return to Office/Building Access Appointments.md]: Existing docs (i.e. in this repo) give all examples via cURL ...

Comment (klondikemarlen) [reply Return to Office/Building Access Appointments.md]: Andrew, Danella and I opted to switch to JS + Axios as it is much more readable and seems to be the "lingua franca" of the web (for this rev of the API docs).

File: Return to Office/Building Access Appointments.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+
+```js
+axios({
+  method: 'POST',
+  url: `${PROOF_API_HOST}/api/building_access_appointments`,
+  headers: {
+    'Accept': 'application/json',
+    'Authorization': `Bearer ${PROOF_API_TOKEN}`,
+    'Content-Type': 'application/json',
+  },
```
Comment (danerio) [inline Return to Office/Building Access Appointments.md]: I'm uncomfortable giving an example for an endpoint that is not supported

Comment (klondikemarlen) [reply Return to Office/Building Access Appointments.md]: It is hidden behind a spoiler tag ...

Comment (klondikemarlen) [reply Return to Office/Building Access Appointments.md]: image

Comment (danerio) [reply Return to Office/Building Access Appointments.md]: Yes, I understand... I stand by my stance to remove the snippet (albeit hidden by a spoiler tag), but... up to you.

**Review by danerio — DISMISSED** [review]

**Review by danerio — APPROVED** [review]

**Pull request merged**